diamond_agent/tasks.py

```diff
@@ -220,8 +220,21 @@ def config_handlers(ctx, handlers, config_path, handlers_path):
     """
     if handlers is None:
         handlers = copy_objects.deepcopy(DEFAULT_HANDLERS)
-        handlers['cloudify_handler.cloudify.CloudifyHandler']['config'][
-            'server'] = get_manager_ip()
+
+        cloudify_agent = ctx.bootstrap_context.cloudify_agent
```

Comment:
The lack of defaults for credentials I just addressed in another comment.

Barring that, are there any cases when we should expect this to fail- I think the only case would be if we support using the cloudify-diamond-plugin without a manager? Do we support that and/or are there other failure scenarios I'm not seeing?